src/components/PublicCommentCard/PublicCommentCard.scss

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of the Digital Planning Register project.
+ *
+ * Digital Planning Register is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Digital Planning Register is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Digital Planning Register. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+@import "src/styles/component-base";
+
+.dpr-public-comment-card {
+  > div {
+    border-left: 5px solid govuk-colour("mid-grey");
+    margin-top: govuk-spacing(7);
+    margin-bottom: govuk-spacing(7);
+  }
+
+  &:after {
+    margin-top: govuk-spacing(7);
+    border-bottom: 1px solid #b1b4b6;
+  }
+
+  &:last-of-type:after {
+    border-bottom: none;
+  }
+
+  &__header {
+    padding: govuk-spacing(0) govuk-spacing(3);
+    box-sizing: border-box;
+    width: 100%;
+  }
+
+  &--toggle {
```

Comment:
Toggle isn't a modifier in the context of this component 

`&__toggle-button` would probably be more accurate?